modular_skyrat/modules/reagent_forging/code/forge_weapons.dm

```diff
@@ -62,15 +118,59 @@
 	. = ..()
 	AddComponent(/datum/component/two_handed, force_unwielded=8, force_wielded=16)
 
+/obj/item/forging/reagent_weapon/bokken
+	name = "reagent bokken"
+	desc = "A bokken that can be imbued with a reagent. It can be dual-wielded to increase block chance!"
+	force = 10
+	icon_state = "bokken"
+	inhand_icon_state = "sword"
+	throwforce = 10
+	block_chance = 30
+	slot_flags = ITEM_SLOT_BACK
+	w_class = WEIGHT_CLASS_NORMAL
+	resistance_flags = FIRE_PROOF
+	attack_verb_continuous = list("bonks", "bashes", "whacks", "pokes", "prods")
+	attack_verb_simple = list("bonk", "bash", "whack", "poke", "prod")
+	var/wielded = FALSE
+
+/obj/item/forging/reagent_weapon/bokken/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text, final_block_chance, damage, attack_type)
+	if(wielded)
+		final_block_chance *= 2
+	if(prob(final_block_chance))
+		if(attack_type == PROJECTILE_ATTACK)
```

Comment:
I **strongly** recommend that projectiles have a very low block chance unless the user is explicitly trained, especially considering PROJECTILE_ATTACK applies to lasers as well (which would melt thru a wooden weapon (another reason not to use bokkens for combat!!!))